src/ToastContainer.js

```diff
@@ -139,8 +139,14 @@ export default class ToastContainer extends Component {
   }
 
   render() {
+    const divProps = Object.assign({}, this.props);
```

Comment:
Please just use `_.omit(this.props, ["toastMessageFactory", "toastType", "preventDuplicates", "newestOnTop"])` since lodash is already included as dependency.